pkg/version/version.go

```diff
@@ -84,7 +83,7 @@ func GetBuild() *Build {
 		BuildDate:     BuildDate,
 		BuildNumber:   BuildNumber,
 		State:         State,
-		PluginVersion: MaxPluginVersion,
+		PluginVersion: feature.MaxPluginVersion,
```

Comment:
(MaxPluginVersion - 1) should be used as current version, because the plugin is used to migrate data from (X-1) to X.
if we use MaxPluginVersion directly here, plugin should work as X => (X+1).

I think current logic is easy to understand, so please keep that, and make sure current version is always same to latest plugin version.